storage/local/series.go

```diff
@@ -504,14 +504,14 @@ func (s *memorySeries) preloadChunksForRange(
 // head returns a pointer to the head chunk descriptor. The caller must have
 // locked the fingerprint of the memorySeries. This method will panic if this
 // series has no chunk descriptors.
-func (s *memorySeries) head() *chunkDesc {
+func (s *memorySeries) head() *ChunkDesc {
 	return s.chunkDescs[len(s.chunkDescs)-1]
 }
 
 // firstTime returns the timestamp of the first sample in the series.
 //
 // The caller must have locked the fingerprint of the memorySeries.
-func (s *memorySeries) firstTime() model.Time {
+func (s *memorySeries) FirstTime() model.Time {
```

Comment:
@mattkanwisher Did you intend to make `FirstTime()` of the `memorySeries` exported as well? Probably a search&replace mistake, since `memorySeries` isn't even public?